docs/2.6.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/participant.rst

```diff
@@ -1,142 +1,94 @@
+
+Exclusive, application-level database locks - tied to the database connection lifetime - enforce the leader election and set the chosen replica as single writer. 
+
+.. NOTE::
+  Alternative approaches for leader election, such as Raft, are unsuitable because the leader status can be lost in between the leader check and the use of the shared resource, i.e. writing to the database. Therefore, we cannot guarantee a single writer.
+
+Exclusive Lock Acquisition
+""""""""""""""""""""""""""
 
-The replicas of a replicated participant node share the same database, which is
-required for two reasons:
-
-- Share the command ID deduplication state of the ledger API command submission
-  service between replicas to prevent double submission of commands in case of
-  fail-over.
-- Obtain consistent ledger offsets across the replicas, otherwise the
-  application could not seamlessly fail-over to another replica. The ledger
-  offsets are decided by the database based on the insertion order of publishing
-  events in the multi-domain event log, i.e., the ledger offset derivation is
-  not deterministic.
-
-Participant Node Replica Monitoring and Fail-Over
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Operating a participant node in a replicated active-passive configuration with a
-shared database requires to establish the active replica, i.e., perform a leader
-election, and to enforce a single writer, i.e., the active replica, to the
-shared database.
-
-We are using exclusive application-level database locks tied to the lifetime of
-the connection to the database to achieve leader election and a enforce single
-writer. Alternative existing approaches for leader election, such as using Raft,
-are not suitable because in between the leader check and the use of the shared
-resource, i.e., writing to the database, the leader status could have been lost
-and we cannot guarantee a single writer.
-
-Leader Election through Exclusive Lock Acquisition
-""""""""""""""""""""""""""""""""""""""""""""""""""
-
-A participant node replica tries to acquire an exclusive application level lock
-(e.g. `Postgres advisory lock
-<https://www.postgresql.org/docs/11/explicit-locking.html#ADVISORY-LOCKS>`_)
-bound to a particular database connection and use that same connection for all
-writes that are not idempotent. The replica that has acquired the lock is the
-leader and the active replica. Using the same connection for writes ensures that
-the lock is held while writes are performed.
+A participant node replica acquires an exclusive application-level lock (e.g. `Postgres advisory lock <https://www.postgresql.org/docs/11/explicit-locking.html#ADVISORY-LOCKS>`_) which is bound to a particular database connection. The active replica that acquires the lock becomes the leader. The replica then uses the same connection for all writes that are not idempotent. 
```

Comment:
slightly outdated: The replica uses a write connection pool that is tied to an exclusive lock on a main connection, and a shared lock on all pool connections. If the main lock is lost, the pool's connections are ramped down. the new active replica needs to wait until all the passive node's pool connections are closed, which is done by trying to acquire the shared lock in exclusive mode